src/components/MapFacetContents.js

```diff
@@ -200,14 +200,24 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
+        let longitude = 0;
```

Comment:
If the values are not set in the bucket, then I don't know if it makes sense to default to (0,0).... I'd prefer to not show the point at all and maybe do a console.warn() to say that something wasn't right... (For geographic facets, the values should always be there, except in cases like the one that caused this bug where the data structure returned by the back end changed...

Reply:
Makes sense. But wouldn't it be incorrect to dismiss a location if it is [0,0]? Even that is a valid location co-ordinate actually!
Should we maybe default the co-ordinates to `NaN` and dismiss them if they are still `NaN` after assignment from value?

Reply:
If it is really (0, 0) we should definitely show it...
If the value for the bucket was "foo" I don't think we should show it...
So, yeah, the NaN thing would work. And Number.parseFloat() will return NaN if the string isn't a number, so that's good.

I'm not sure what React will do if the `points` array contains some null values. If that works and React just ignores them, like if you put `{null}` somewhere in your JSX, then that's fine. Otherwise, you might need to use `forEach()` instead of `map()` and only push the <Marker> components onto the `points` array, skipping the ones that are invalid.. (In actuality, I would expect that if one is invalid, they'll all be invalid, but it still can't hurt..)

Reply:
`points` is directly injected as child of `<Map>` and react seems to ignore `null` values in `points`. So, the `NaN` logic should work out :+1: